src/test/kotlin/com/jayrave/moshi/pristineModels/IntegrationTest.kt

```diff
@@ -0,0 +1,71 @@
+package com.jayrave.moshi.pristineModels
+
+import com.jayrave.moshi.pristineModels.testLib.readFileContent
+import com.squareup.moshi.Moshi
+import com.squareup.moshi.Types
+import org.assertj.core.api.Assertions
+import org.junit.Test
+import java.lang.reflect.Type
+
+class IntegrationTest {
```

Comment:
I haven't made a big distinction between unit & integration tests in this work. I think this can be moved in one of the test cases in `MapperTest.kt`

Reply:
None of the existing test `MapperTest` class have references to `PristineModelsJsonAdapterFactory` and vice versa. So just wanted to put it in a separate file. I can move it.